ndbench-janusgraph-plugins/src/main/java/com/netflix/ndbench/plugins/janusgraph/JanusGraphPlugin.java

```diff
@@ -0,0 +1,105 @@
+package com.netflix.ndbench.plugins.janusgraph;
```

Comment:
To match the naming convention of Janusgraph we may want to name the plugin `JanusGraphPluginCQL` or some relevant name such that people can add more plugins that are not CQL specific.

Reply:
This class is independent of what backend JanusGraph uses (cql, bigtable, etc). The code that does the backend wiring is in JanusGraphBuilderProviderProxy

Reply:
I understand that `JanusGraphBuilderProviderProxy` does the wiring part but when you are in the NDBench UI and you are trying to select a plugin out of the many plugins under Janusgraph, you have to have some knowledge that this is CQL. Otherwise, how will you be able to run two NDBench clusters one against CQL and one against another storage engine in the future?

Reply:
I did not expect to run the benchmark with different storage backends simultaneously. Hence, one can alter the config to choose the backend being tested. What you suggested is valid. Let me address it. thanks!